components/google-cloud/containers/uCAIP_MB_SDK/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the \"License\");
+# you may not use this file except in compliance with the License.\n",
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an \"AS IS\" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Base image to use for this docker
+FROM gcr.io/google-appengine/python
```

Comment:
Meanwhile is there a reason not using the public Python image hosted on dockerhub?

Reply:
done for adding `latest`. 
Leaving this comment open while following up on using public image versus google owned image.

Reply:
regarding using gcr hosted versus docker-hub; discussed this offline with the team, I think we prefer to stay with the google hosted image since these components are for google cloud only in case there is a deviation between appengine and public image.